resources/views/components/nav-bar.blade.php

```diff
@@ -1,19 +1,23 @@
 <nav class="navbar w-100  py-1 ps-2 pe-3 bg-white shadow-sm" >
     <div class="nav-left d-flex align-items-center gap-2">
         <button id="sidebar-toggle"  href="" class="btn btn-nav m-0 p-0">
-            <svg xmlns="http://www.w3.org/2000/svg"  width="25" height="25" fill="currentColor" class="bi bi-justify" viewBox="0 0 16 16">
+            <svg xmlns="http://www.w3.org/2000/svg"  width="20" height="20" fill="currentColor" class="bi bi-justify" viewBox="0 0 16 16">
                 <path fill-rule="evenodd" d="M2 12.5a.5.5 0 0 1 .5-.5h11a.5.5 0 0 1 0 1h-11a.5.5 0 0 1-.5-.5m0-3a.5.5 0 0 1 .5-.5h11a.5.5 0 0 1 0 1h-11a.5.5 0 0 1-.5-.5m0-3a.5.5 0 0 1 .5-.5h11a.5.5 0 0 1 0 1h-11a.5.5 0 0 1-.5-.5m0-3a.5.5 0 0 1 .5-.5h11a.5.5 0 0 1 0 1h-11a.5.5 0 0 1-.5-.5"/>
             </svg>
         </button>
-
-        <form class="" role="search" >
-            <input class="form-control me-5  rounded-5" style="height: 25px" type="search" placeholder="Search" aria-label="Search">
-        </form>
+        <h1 class="fs-5" >{{ $title }}</h1>
     </div>
 
     <div class="nav-right d-flex align-items-center gap-2 ">
-        <svg xmlns="http://www.w3.org/2000/svg" width="25" height="25" fill="#022658" class="bi bi-bell" viewBox="0 0 16 16">
-            <path d="M8 16a2 2 0 0 0 2-2H6a2 2 0 0 0 2 2M8 1.918l-.797.161A4 4 0 0 0 4 6c0 .628-.134 2.197-.459 3.742-.16.767-.376 1.566-.663 2.258h10.244c-.287-.692-.502-1.49-.663-2.258C12.134 8.197 12 6.628 12 6a4 4 0 0 0-3.203-3.92zM14.22 12c.223.447.481.801.78 1H1c.299-.199.557-.553.78-1C2.68 10.2 3 6.88 3 6c0-2.42 1.72-4.44 4.005-4.901a1 1 0 1 1 1.99 0A5 5 0 0 1 13 6c0 .88.32 4.2 1.22 6"/>
-        </svg>
+        <a href="{{ route('logout') }}" onclick="event.preventDefault(); document.getElementById('logout-form').submit();" class="rounded-5 border-black border-1" title="Logout">
+            <svg xmlns="http://www.w3.org/2000/svg" width="22" height="22" fill="#022658" class="bi bi-box-arrow-right" viewBox="0 0 16 16">
```

Comment:
For icons use `webfont` icons, instead of SVG icons. replace all SVG icons into Webfont icons